# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fastfetch/hotfix-hppa.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fastfetch-2.53.0/src/detection/cpu/cpu_linux.c.vanilla	2025-10-04 18:21:29.994000000 +0200
+++ fastfetch-2.53.0/src/detection/cpu/cpu_linux.c	2025-10-04 18:21:42.130330324 +0200
@@ -356,6 +356,8 @@
             (cpu->name.length == 0 && ffParsePropLine(line, "processor 0:", &cpu->name)) ||
             (cpu->vendor.length == 0 && ffParsePropLine(line, "vendor_id :", &cpu->vendor)) ||
             (cpuMHz->length == 0 && ffParsePropLine(line, "cpu MHz static :", cpuMHz)) || // This one cannot be detected because of early return
+            #elif __hppa__
+            (cpu->name.length == 0 && ffParsePropLine(line, "cpu :", &cpu->name)) ||
             #elif __ia64__
             (cpu->name.length == 0 && ffParsePropLine(line, "model name :", &cpu->name)) ||
             (cpu->vendor.length == 0 && ffParsePropLine(line, "vendor :", &cpu->vendor)) ||
